src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandOddsRatio.java

```diff
@@ -20,37 +20,100 @@
  *
  * <h3>Statistical notes</h3>
- * <p> Odds Ratios in the 2x2 contingency table below are</p>
  *
- * $$ R = \frac{X[0][0] * X[1][1]}{X[0][1] * X[1][0]} $$
- *
- * <p>and its inverse:</p>
+ * <p>The following 2x2 contingency table gives the notation for allele support and strand orientation.</p>
  *
  * <table>
- *      <tr><td>&nbsp;</td><td>+ strand </td><td>- strand</td></tr>
- *      <tr><td>REF;</td><td>X[0][0]</td><td>X[0][1]</td></tr>
- *      <tr><td>ALT;</td><td>X[1][0]</td><td>X[1][1]</td></tr>
+ * <tr><th>&nbsp;</th><th>+ strand&nbsp;&nbsp;&nbsp;</th><th>- strand&nbsp;&nbsp;&nbsp;</th></tr>
+ * <tr><th>REF&nbsp;&nbsp;&nbsp;</th><td>X[0][0]</td><td>X[0][1]</td></tr>
+ * <tr><th>ALT&nbsp;&nbsp;&nbsp;</th><td>X[1][0]</td><td>X[1][1]</td></tr>
  * </table>
  *
- * <p>The sum R + 1/R is used to detect a difference in strand bias for REF and for ALT (the sum makes it symmetric). A high value is indicative of large difference where one entry is very small compared to the others. A scale factor of refRatio/altRatio where</p>
+ * <p>We can then represent the Odds Ratios with the equation:</p>
+ *
+ * <img src="http://latex.codecogs.com/svg.latex?$$ R = \frac{X[0][0] * X[1][1]}{X[0][1] * X[1][0]} $$" border="0"/>
+ *
+ * <p>and its inverse:</p>
+ *
+ * <img src="http://latex.codecogs.com/svg.latex?$$ \frac{1}{R} = \frac{X[0][1] * X[1][0]}{X[0][0] * X[1][1]} $$" border="0"/>
  *
- * $$ refRatio = \frac{max(X[0][0], X[0][1])}{min(X[0][0], X[0][1} $$
+ * <p>The sum R + 1/R is used to detect a difference in strand bias for REF and for ALT. The sum makes it symmetric.
+ * A high value is indicative of large difference where one entry is very small compared to the others. A scale factor
+ * of refRatio/altRatio where</p>
+ *
+ * <img src="http://latex.codecogs.com/svg.latex?$$ refRatio = \frac{min(X[0][0], X[0][1])}{max(X[0][0], X[0][1])} $$" border="0"/>
  *
  * <p>and </p>
  *
- * $$ altRatio = \frac{max(X[1][0], X[1][1])}{min(X[1][0], X[1][1]} $$
+ * <img src="http://latex.codecogs.com/svg.latex?$$ altRatio = \frac{min(X[1][0], X[1][1])}{max(X[1][0], X[1][1])} $$" border="0"/>
+ *
+ * <p>ensures that the annotation value is large only. The final SOR annotation is given in natural log space.</p>
+ *
+ * <p>See the <a href="http://www.broadinstitute.org/gatk/guide/article?id=4732">method document on statistical tests</a>
+ * for a more detailed explanation of this statistical test.</p>
+ *
+ * <h3>Example calculation</h3>
+ *
+ * <p>Here is a variant record where SOR is 0.592.</p>
+ *
+ * <pre>
+ *     AC=78;AF=2.92135e-02;AN=2670;DP=31492;FS=48.628;MQ=58.02;MQRankSum=-2.02400e+00;MQ_DP=3209;QD=3.03; \
+ *     ReadPosRankSum=-1.66500e-01;SB_TABLE=1450,345,160,212;SOR=0.592;VarDP=2167
+ * </pre>
+ *
+ * <p>Read support shows some strand bias for the reference allele but not
+ * the alternate allele. The SB_TABLE annotation (a non-GATK annotation) indicates 1450 reference alleles on the forward strand, 345
```

Comment:
Is it useful to include my example verbatim with the "non-GATK annotation"? 

1450 reference alleles -> 1450 reference reads
ditto alternate alleles -> reads

Reply:
This is the example you gave and so this is the example I used to illustrate the calculation. I apologize I cannot spend any more time on this even if I want to. I have ~ a week left to finalize the gCNV tutorial. Please ask @rcmajovski, who owns GATK documentation, for additional Comms support.